google-cloud-spanner/src/test/java/com/google/cloud/spanner/SpannerOptionsTest.java

```diff
@@ -63,4 +70,13 @@ public void builder() {
     assertThat(options.getProjectId()).isEqualTo(projectId);
     assertThat(options.getPrefetchChunks()).isEqualTo(2);
   }
+
+  class DummyTransportOptions implements TransportOptions {
```

Comment:
In Spanner in particular I get the following:

`java.lang.ClassNotFoundException: com.google.auth.http.HttpTransportFactory`

I didn't want to mess around with dependencies, so this seemed like the easiest way to go.